### DISCUSSION TITLE
Change the spacing of elements

### DISCUSSION THREAD

**Discussion by joffreypersia** [tailwindlabs/tailwindcss#11162] [Ideas]
I was surprised that we cannot change 'only' the spacing of different items. 
Like `prose-sm` doest a weird behavior where it shrink the spaces but also the heading sizes. 

Is there a way to improve the liberty of only changing the spaces between items
like 
`prose-space-s` / md / l / xl

Thank you for this great piece of tool :)

**Comment by wongjn** [comment]
Yep! You can use a Tailwind plugin or custom CSS to create rules that adjust the spacing between elements.

**Reply by joffreypersia** [reply]
Hi, 

Thank you for your answer. 

Can you develop more ? 
Which plugin is build for adjusting the prose spacing ? 

I would rather not just overlap the current style, but just replace / modify the values.

**Reply by wongjn** [reply]
> Can you develop more ?

I'm sorry, but I cannot quite ascertain the meaning behind this inquiry.

> Which plugin is build for adjusting the prose spacing ?

I presume you could roll your own if an existing one does not exist.

**Reply by joffreypersia** [reply]
Ah ok you mean building my own plugin or look for a custom one build by someone. 

I was maybe trying to push the dev to develop this tool a little bit more :)

Best,
Joffrey
